examples/umd-client/index.html

```diff
@@ -18,7 +18,7 @@
     <script src="https://unpkg.com/styled-components@4.1.3/dist/styled-components.min.js" crossorigin></script>
 
     <!-- Load Kepler.gl-->
-    <script src="https://unpkg.com/kepler.gl@3.0.0-alpha.0/umd/keplergl.min.js"></script>
+    <script src="https://unpkg.com/kepler.gl/umd/keplergl.min.js"></script>
```

Comment:
Q: Have you confirmed if this load the latest 2.x production version or the latest alpha version? I am assuming the alpha versions are published with a tag and I seem to remember that one needs to specify the tag in the unpkg url in such cases?

I assume we want this to load the same version that is hosted on kepler.gl/demo.

@igorDykhta Any thoughts?

Reply:
When I tested this locally it loaded 2.5.5. I also browsed the available versions at [https://unpkg.com/browse/kepler.gl/](https://unpkg.com/browse/kepler.gl/)

I can see alpha tags for previous versions but not 3.0.0. https://kepler.gl/demo/ appears to be all compiled inside a `bundle.js` , so I couldn't double check the version (though I see it on the browser on the left-hand side)

My thinking was that this change would mean as soon as 3.0 _is_ published it would be used, and in the meantime the latest version would be used which seems to be the workaround working for most folks based on past issues filed.

Reply:
confirming this works if you change the version of kepler.gl to load:

```html
<script src="https://unpkg.com/kepler.gl@2.5.5/umd/keplergl.min.js" crossorigin></script>
```